public/api/conf/2.0/schemas/wco/fileupload/FileUploadResponse.xsd

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns="http://gov.uk/customs/declarations/file-upload" xmlns:xs="http://www.w3.org/2001/XMLSchema"
```

Comment:
in `FileUploadRequest` i have `targetNamespace="hmrc:fileupload2"` so these are now inconsistent. I think they should be the same, and different from the current one